third_party/move/move-bytecode-verifier/src/dependencies.rs

```diff
@@ -262,7 +262,8 @@ fn verify_imported_structs(context: &Context) -> PartialVMResult<()> {
                         StatusCode::TYPE_MISMATCH,
                         IndexKind::StructHandle,
                         idx as TableIndex,
-                    ));
+                    )
+                    .with_message("imported struct mismatches expectation"));
```

Comment:
I am assuming it is backward compatible to add these messages (likely because they are stripped off before storing on chain)?